apps/docs/src/app/platform/component-docs/platform-forms/checkbox/platform-checkbox-header/platform-checkbox-header.component.html

```diff
@@ -0,0 +1,30 @@
+<header>Checkbox</header>
+<description>
+    Platform Checkbox is wrapper implemention on
```

Comment:
I was under impression that we have new docs that we try to polish in gitbook but I can see old  text here.

Reply:
have reviewed documentation with Manju and updated it now.